README.md

```diff
@@ -1,2 +1,57 @@
 # flagsmith-openfeature-provider-python
-Flagsmith OpenFeature provider for the python SDK
+
+The Flagsmith provider allows you to connect to your Flagsmith instance through the OpenFeature SDK
+
+## Python SDK usage
+
+### Install dependencies
+
+First, you'll need to install the OpenFeature SDK and the Flagsmith Provider.  
+
+```bash
+pip install openfeature-sdk openfeature-provider-flagsmith
+```
+
+### Using the Flagsmith Provider with the OpenFeature SDK
+
+To create a Flagsmith provider you will need to provide a number of arguments. These are shown and described 
+below. See the [Flagsmith docs](https://docs.flagsmith.com/clients/server-side) for further information on the 
+configuration options available for the Flagsmith python client.
+
+```python
+from flagsmith import Flagsmith
+from openfeature_flagsmith.provider import FlagsmithProvider
+
+provider = FlagsmithProvider(
+    # Provide an instance of the Flagsmith python client.
+    # Required: True
+    client=Flagsmith(...),
+    
+    # By enabling the use_flagsmith_defaults setting, you can instruct the OpenFeature SDK to use
+    # the default logic included in the Flagsmith client. This will override the default provided
+    # at evaluation time in the OpenFeature SDK in most cases (excluding those where an unexpected
+    # exception happens in the Flagsmith client itself).
+    # Required: False
+    # Default: False
+    use_flagsmith_defaults=False,
+    
+    # By default, when evaluating the boolean value of a feature, the Flagsmith OpenFeature Provider
+    # will use the 'Enabled' state of the flag. This behaviour can be changed to use the value
```

Comment:
What is "value" here?

Reply:
Not sure how to make this more clear tbh, it's referring to the 'value' field in the Flagsmith feature: 

<img width="191" alt="image" src="https://github.com/Flagsmith/flagsmith-openfeature-provider-python/assets/14089968/a02f39f4-f70c-4a9e-883d-fa3617da897d">

Any suggestions?

Reply:
I've tried my best to reword it so it's clearer.